external-crates/move/crates/move-compiler/tests/custom_rules/combinable_bool.move

```diff
@@ -0,0 +1,25 @@
+module 0x42::M {
```

Comment:
Please follow the pattern for tests we arrived at for the constant naming 
- Correct `linter` dir
- Separate positive/negative tests
- a case to the allow(lint(_)) case

Reply:
refactored